package.json

```diff
@@ -5,7 +5,6 @@
   "homepage": "/sosialhjelp/soknad/",
   "dependencies": {
     "@sentry/browser": "^5.15.5",
-    "babel-polyfill": "^6.26.0",
```

Comment:
Fungerer det fortsatt på ie uten denne?

Reply:
Jepp. core-js har tatt over for babel-polyfill. https://github.com/zloirock/core-js#babelpolyfill Det eneste jeg ikke blir klok på er om vi nå bør importere `regenerator-runtime` også. 

Jeg har trykket gjennom søknaden i IE11 på labs, og der fungerer alt fint.

Reply:
Jeg har googlet litt mer, og det ser ut til at Babel også faser ut bruken av `regenerator-runtime`. https://babeljs.io/docs/en/v7-migration#remove-proposal-polyfills-in-babel-polyfill-https-githubcom-babel-babel-issues-8416

Det ser heller ikke ut til at vi har brukt `babel-polyfill` på en god stund (vi inkluderer `core-js` direkte), så det bør være trygt å flette inn denne. :)